addons/color-palette/PaletteImporter.gd

```diff
@@ -2,11 +2,13 @@ tool
 class_name PaletteImporter
 extends Reference
 
-
 # Adapted from Github -> Orama-Interactive/Pixelorama/src/Autoload/Import.gd
 static func import_gpl(path : String) -> Palette:
+	var color_line_regex = RegEx.new()
+	color_line_regex.compile("(?<red>[0-9]{1,3})[ \t]+(?<green>[0-9]{1,3})[ \t]+(?<blue>[0-9]{1,3})[ \t]+(?<hex>[0-9a-fA-F]{6})")
```

Comment:
The HEX is not actually part of the GPL format, the thing after the 3 numbers is just an optional name, see the gimp source here:

https://gitlab.gnome.org/GNOME/gimp/-/blob/gimp-2-10/app/core/gimppalette-load.c#L149-176

This means if your palette does not include the hex at the end it will fail:
https://regexr.com/65t74

Reply:
Makes sense! I removed the hex bit of the regex entirely as it is not used anyway.